src/Timeline.tsx

```diff
@@ -52,12 +52,19 @@ type Animation =
       toDomain: Domain
     }>
 
+const defaultTimeMin = Date.now() - 100000
```

Comment:
Should this be configurable?

Reply:
I made it a parameter to make it more configurable now